spec/rails_helper.rb

```diff
@@ -93,7 +93,6 @@ class Analytics
 
   config.before(:each) do
     DocAuth::Mock::DocAuthMockClient.reset!
-    original_queue_adapter = ActiveJob::Base.queue_adapter
```

Comment:
Maybe the original intention was to tear down based on what's overridden below with the `ActiveJob::Base.queue_adapter = :inline` ? Do we need that?